crates/core/src/syn/parser/prime.rs

```diff
@@ -39,6 +39,16 @@ impl Parser<'_> {
 			t!("u\"") | t!("u'") | TokenKind::Glued(Glued::Uuid) => {
 				Ok(Value::Uuid(self.next_token_value()?))
 			}
+			t!("f\"") | t!("f'") | TokenKind::Glued(Glued::File) => {
+				if !self.settings.files_enabled {
+					bail!("Cannot create a file pointer, as the experimental files capability is not enabled", @self.last_span);
```

Comment:
For consistency the error here should use the `unexpected!` like other places where syntax is disabled because of experimental flags.